slo_generator/backends/cloud_monitoring.py

```diff
@@ -220,20 +225,28 @@ def get_window(timestamp, window):
         Returns:
             :obj:`monitoring_v3.types.TimeInterval`: Measurement window object.
         """
-        measurement_window = monitoring_v3.types.TimeInterval()
-        measurement_window.end_time.seconds = int(timestamp)
-        measurement_window.end_time.nanos = int(
-            (timestamp - measurement_window.end_time.seconds) * 10**9)
-        measurement_window.start_time.seconds = int(timestamp - window)
-        measurement_window.start_time.nanos = measurement_window.end_time.nanos
+        end_time_seconds = int(timestamp)
```

Comment:
Same here. I took these snippets from the official repo:
- https://github.com/googleapis/python-monitoring/blob/main/samples/snippets/v3/cloud-client/quickstart.py
- https://github.com/googleapis/python-monitoring/blob/main/samples/snippets/v3/cloud-client/snippets.py